src/analysis/complex/exponential.lean

```diff
@@ -3,7 +3,9 @@ Copyright (c) 2018 Chris Hughes. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes, Abhimanyu Pallavi Sudhir, Jean Lo, Calle Sönne
 -/
-import tactic.linarith data.complex.exponential analysis.specific_limits
+import tactic.linarith
+import data.complex.exponential
+import analysis.specific_limits
       group_theory.quotient_group analysis.complex.basic
```

Comment:
I guess your script has been trapped here...

Reply:
Oh, I've missed lots. :-(

Reply:
I better go to sleep, but I'll think about another regex tomorrow.

Reply:
I've put an `import` on every indented line. So now your original script should be able to do the rest.

I used
```
find . | grep "[.]lean$" | xargs sed -i '/^import/,+1 s/^ [ ]*/import /'
```
which I ran 5 times as a goofy manual while-loop (-;